desktop/components/current_auctions/utils/upcoming_label.js

```diff
@@ -0,0 +1,24 @@
+import moment from 'moment'
+import { include } from 'underscore'
+
+function isPreviewState (status) {
```

Comment:
Sorry I didn't catch this earlier, but looks like we can also get the `preview` state from metaphysics: https://github.com/artsy/metaphysics/blob/master/schema/sale/index.js#L169